bigbluebutton-html5/imports/api/presentations/server/handlers/presentationConversionUpdate.js

```diff
@@ -61,8 +61,8 @@ export default function handlePresentationConversionUpdate({ body }, meetingId)
     case GENERATED_SLIDE_KEY:
       statusModifier['conversion.pagesCompleted'] = body.pagesCompleted;
       statusModifier['conversion.numPages'] = body.numberOfPages;
+      if (body.pagesCompleted === body.numberOfPages) statusModifier['conversion.done'] = true;
```

Comment:
Was the "conversion done" not triggered from bbb-web? It looks odd to have to check this on client side

Reply:
In a new debugging I've found out that bbb-web is properly triggering 'conversion done'. So, we can disregard this line of code for now.